packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -285,22 +285,6 @@ class FlutterPlugin implements Plugin<Project> {
         return "release"
     }
 
-    /**
-     * Returns a Flutter Jar file path suitable for the specified Android buildMode.
-     */
-    private File flutterJarFor(buildMode) {
-        if (buildMode == "profile") {
-            return profileFlutterJar
```

Comment:
These `...FlutterJar` variables pop up a lot throughout the rest of the script. It looks like we're still setting them as a dependency and trying to find them, but now we're never using them. Can we remove them from the rest of this script too?